### ISSUE TITLE
Does not work with holochain v0.3.0-beta-dev.23

### ISSUE THREAD

**Issue by guillemcordoba** [holochain/holochain-client-js#211]
While trying to upgrade to that version of holochain, @holochain/client stops working altogether. Any call that it tries to make, results in this kind of error being returned:

```
{
  type: { deserialization: null },
  data: 'Bytes(Deserialize("wrong msgpack marker FixStr(11)"))'
}
```

This happens with the `AdminInterface` as well as with the `AppInterface`. 

Maybe this is not an issue with the client... I've spent a lot of time exploring its causes with no result. I did find [this issue](https://github.com/3Hren/msgpack-rust/issues/336), and also noticed that now the holochain git repository depends on two versions of rmp-serde, 0.15 and 1.1 - maybe that's the cause? But I don't think so because I _was_ able to run sweettests without problems.

Maybe this PR was the one that introduced the problem? https://github.com/holochain/holochain/pull/2844

**Comment by jost-s** [comment]
Yes, the ser/de changed somehow. I'll release a matching client for the time being and look into reverting this change on the Holochain side. It's not for the better IMO, from
```
{
  type: "error",
  data: ...
}
```
to
```
  type: { error: null },
  data: ...
}
```

**Comment by jost-s** [comment]
fixed by #212
tryorama release to follow
